src/docfx/build/xref/XrefResolver.cs

```diff
@@ -44,8 +44,8 @@ internal class XrefResolver
             SourceInfo<string> href, Document hrefRelativeTo, Document inclusionRoot)
         {
             var (uid, query, fragment) = UrlUtility.SplitUrl(href);
-            string? moniker = null;
-            string? text = null;
+            var moniker = default(string);
```

Comment:
does this really look better :)

Reply:
Maybe some folks like to see `var` alignments. (P.S., I'm one of them, but I didn't tell @mingwli to make this change 🤣 )